src/agent/block_store_services/block_store_mongo.js

```diff
@@ -112,7 +112,7 @@ class BlockStoreMongo extends BlockStoreBase {
                     dbg.log0('found usage data usage_data = ', this._usage);
                 }
             })
-            .then(() => this.set_mongo_store_limit())
+            //.then(() => this.set_mongo_store_limit())
```

Comment:
better remove the line + it looks like it is the only place that set_mongo_store_limit() so we can also remove the function itself.